src/Products/CMFCore/tests/test_CMFCatalogAware.py

```diff
@@ -194,9 +194,9 @@ def test_reindexObjectSecurity(self):
         foo.reindexObjectSecurity()
         log = sorted(cat.log)
         self.assertEqual(log, [
-            'reindex /site/foo %s 1' % str(CMF_SECURITY_INDEXES),
-            'reindex /site/foo/bar %s 1' % str(CMF_SECURITY_INDEXES),
-            'reindex /site/foo/hop %s 1' % str(CMF_SECURITY_INDEXES),
+            'reindex /site/foo %s 0' % str(CMF_SECURITY_INDEXES),
+            'reindex /site/foo/bar %s 0' % str(CMF_SECURITY_INDEXES),
+            'reindex /site/foo/hop %s 0' % str(CMF_SECURITY_INDEXES),
```

Comment:
This seems wrong: even with `update_metadata=0` I would expect that the `CMF_SECURITY_INDEXES` are reindexed for `foo` and children -- at least if something has changed regarding security for `foo` or its children.

If nothing has changed regarding security of those objects, we want a test for the case when something regarding security has changed.

Reply:
> This seems wrong: even with `update_metadata=0` I would expect that the `CMF_SECURITY_INDEXES` are reindexed for `foo` and children -- at least if something has changed regarding security for `foo` or its children.

I have misinterpreted the digits as numbers (counting index requests). But (almost surely) the messages used by the test simply record reindex requests and the digits represent boolean values corresponding to the `update_metadata` parameter. Thus, the test is correct.